hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/CachedEntryQueue.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.hadoop.hbase.io.hfile.BlockCacheKey;
 import org.apache.yetus.audience.InterfaceAudience;
 
+import org.apache.hbase.thirdparty.com.google.common.base.Preconditions;
```

Comment:
All the changes in this file seem extraneous?
Perhaps the findbugs exclusion can make a difference in the precommit report, but overall these changes come from somewhere other than the thirdparty changes?

Reply:
This is also part of the backport only ([here](https://github.com/apache/hbase/commit/e53712ae993248594245363fc3e8e36022534d9a#diff-e042cf5e7ce8c32da10eb51e88ec6aaf633e4b9a80ecd043b8d987dc0caf4ba9R28))